packages/components/src/date-time/date.js

```diff
@@ -33,7 +46,9 @@ class DatePicker extends Component {
 	 * This focus loss closes the date picker popover.
 	 * Ideally we should add an upstream commit on react-dates to fix this issue.
 	 */
-	keepFocusInside() {
+	keepFocusInside( newMonthDate ) {
```

Comment:
`keepFocusInside()` isn't currently specific to month clicks or changes. Can we keep it as-is and create a new function for month clicks that references it?

Reply:
mmm ok 👍.
In that case, we'll have to create an intermediate function in between the functions bound to the `onPrevMonthClick` and `onNextMonthClick` properties, and the `keepFocusInside` method.

Reply:
Nice thanks! 🙌